shepherd/Goal.py

```diff
@@ -1,41 +1,63 @@
-"""
+import Utils.py
```

Comment:
include newline after this. We might want to from Utils.py import * eventually too.